
# Review: More for loops examples

This reading contains the code used in the instructional videos from [**More for loops examples**](https://www.coursera.org/learn/python-crash-course/lecture/00vRH/more-for-loop-examples)

## **Introduction**

This follow-along reading is organized to match the content in the video that follows. It contains the same code shown in the next video. These code blocks will provide you with the opportunity to see how the code is written, allow you to practice running it, and can be used as a reference to refer back to.

You can follow along in the reading as the instructor discusses the code or review the code after watching the video.


In [1]:

product = 1
for n in range(1,10):
  product = product * n

print(product)


362880




## About this code

This code first defines a variable product and initializes it to 1. Then, it uses a for loop to iterate over the range of numbers from 1 to 9. In each iteration of the loop, the variable n is assigned the current number in the range. The code then multiplies product by n. This means that after the first iteration, product will be equal to 1 * 1 = 1. After the second iteration, product will be equal to 1 * 2 = 2. And so on.

The loop will terminate when the value of n is 9. At this point, the value of product will be 1 * 2 * 3 * ... * 9 = 362880.


In [2]:

def to_celsius(x):
  return (x-32)*5/9

for x in range(0,101,10):
  print(x, to_celsius(x))
    

0 -17.77777777777778
10 -12.222222222222221
20 -6.666666666666667
30 -1.1111111111111112
40 4.444444444444445
50 10.0
60 15.555555555555555
70 21.11111111111111
80 26.666666666666668
90 32.22222222222222
100 37.77777777777778
